draft-ietf-tls-tls13.md

```diff
@@ -499,6 +499,36 @@ informative:
        -
          ins: H. Krawczyk
 
+  MHJT14:
+       title: "I Know Why You Went to the Clinic: Risks and Realization of HTTPS Traffic Analysis"
+       date: 2014
+       seriesInfo: Privacy Enhancing Technologies. PETS 2014
+       target: http://link.springer.com/chapter/10.1007/978-3-319-08506-7_8
+       author:
+       -
+         ins: B. Miller
+       -
+         ins: L. Huang
+       -
+         ins: A. Joseph
+       -
+         ins: D. Tygar
+
+  HCJC16:
```

Comment:
Note that you can use the DOI directly in the latest kramdown-rfc2629. Just cite `{{?HCJ16=DOI.10.1186/s13635-016-0030-7}}` inline and the tools should create a citation for you.